### PR TITLE
docs: add scene docs

### PR DESCRIPTION
**Pull request by gz65555**
- add scene sky docs
- add scene ambientLight docs

### CODE REVIEW

File: docs/light.zh-CN.md [hunk 1]
```diff
@@ -21,13 +21,9 @@ type: 组件
...
-let lightEntity = rootEntity.createChild('ambient_light');
-
-// 创建一个环境光组件
-let ambientLight = lightEntity.addComponent(AmbientLight);
-ambientLight.color = new Color(1, 0, 0);
-ambientLight.intensity = 0.5;
+// 在场景中设置环境光
+const scene = engine.sceneManager.activeScene;
+scene.ambientLight.diffuseSolidColor.setValue(1, 1, 1, 1);
 ```
```
Comment (eyworldwide) [inline docs/light.zh-CN.md:27]: Could you add a playground here?

File: docs/scene.zh-CN.md [hunk 2]
```diff
@@ -34,44 +34,71 @@ engine.sceneManager.activeScene = scene1;
...
+// 添加纯色背景
+background.mode = BackgroundMode.SolidColor; // 默认纯色背景
+background.solidColor.setValue(1, 1, 1, 1); // 纯白色
+
+// 添加天空盒背景
+background.mode = BackgroundMode.Sky; // 默认纯色背景
+const skyMaterial = (background.sky = new SkyBoxMaterial(engine)); // 添加天空盒材质
+skyMaterial.textureCubeMap = textureCube; // 设置立方体纹理
+background.sky.mesh = PrimitiveMesh.createCuboid(engine, 2, 2, 2); // 设置天空盒网格
+```
```
Comment (eyworldwide) [inline docs/scene.zh-CN.md:84]: Could you add a playground here?

File: docs/scene.zh-CN.md [hunk 3]
```diff
@@ -34,44 +34,71 @@ engine.sceneManager.activeScene = scene1;
...
+
+#### 5. 设置场景环境光
+
+场景环境光（AmbientLight）设置：
+
+```typescript
+const scene = engine.sceneManager.activeScene;
+scene.ambientLight.diffuseSolidColor.setValue(1, 1, 1, 1);
+
 ```
```
Comment (eyworldwide) [inline docs/scene.zh-CN.md:94]: Could you add a playground here?

File: src/components/home/Spine.tsx [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
   const scene = engine.sceneManager.activeScene;
   const rootEntity = scene.createRootEntity();
 
   // camera
-  const cameraEntity = rootEntity.createChild("camera_node");
-  const camera = cameraEntity.addComponent(Camera);
-  camera.backgroundColor = new Vector4(51/255, 51/255, 51/255, 1);
+  const cameraEntity = rootEntity.createChild('camera_node');
+  cameraEntity.addComponent(Camera);
+  scene.background.solidColor.setValue(51 / 255, 51 / 255, 51 / 255, 1);
```
Comment (eyworldwide) [inline src/components/home/Spine.tsx:28]: The display of background color with the same value is different from v0.3. See the 2D demo in homepage of site.

File: docs/scene.zh-CN.md (hunk 2 above)
Comment (gz65555) [reply docs/scene.zh-CN.md:84]: Added.

**Review by eyworldwide — APPROVED** [review]
+1

**Pull request merged**